src/gateways/BeaconsGateway.test.ts

```diff
@@ -67,4 +73,34 @@ describe("BeaconsGateway", () => {
       await expect(gateway.getAllBeacons()).rejects.toThrow();
     });
   });
+
+  describe("saveBeacon()", () => {
+    beforeEach(() => {
+      jest.clearAllMocks();
+    });
+
+    it("sends a PATCH request to the correct endpoint", () => {
+      const gateway = new BeaconsGateway(beaconResponseMapper);
+      const updatedFieldsOnly: Partial<IBeacon> = { manufacturer: "ACME Inc." };
+      // @ts-ignore
```

Comment:
This `//@ts-ignore` is necessary because TSlint doesn't know that Jest is mocking axios, so it flags `mockResolvedValue` does not exist on object of type `axios.patch`.